src/devices/src/virtio/balloon/mod.rs

```diff
@@ -47,16 +53,28 @@ const VIRTIO_BALLOON_S_HTLB_PGFAIL: u16 = 9;
 pub enum Error {
     /// Activation error.
     Activate(super::ActivateError),
+    /// No balloon device found.
+    DeviceNotFound,
+    /// Device was not activated because the driver is missing.
+    DriverNotFound,
```

Comment:
nitish:
```suggestion
    /// Device not activated yet.
    DeviceNotActive,
```

I would avoid to imply knowing _why_ the guest hasn't activated the device (yet).